aten/src/ATen/BatchingRegistrations.cpp

```diff
@@ -1082,7 +1081,7 @@ TORCH_LIBRARY_IMPL(aten, Batched, m) {
   UNARY_POINTWISE(ceil);
   UNARY_POINTWISE(cos);
   UNARY_POINTWISE(cosh);
-  UNARY_POINTWISE(_conj);
+  UNARY_POINTWISE(conj);
```

Comment:
Can you add the following:
```
UNARY_POINTWISE(conj_physical)
```

Also, `conj` is technically a view, so we should move it below with all of the complex view functions (see next comment)